### PR TITLE
AVX patch for i40e dpdk-18.02 driver

### PR DESCRIPTION
**Pull request by ajamshed**
Replaces the old dpdk-18.02 AVX i40e patch with official DPDK's version.

Fixes: #34

Signed-off-by: Muhammad Asim Jamshed <muhammad.jamshed@intel.com>

### CODE REVIEW

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

File: install.sh [hunk 1]
```diff
@@ -201,7 +201,7 @@ download_dpdk_zip()
 
 	echo ""
 	echo "Applying AVX not supported patch for resolved dpdk-18.02 i40e driver issue.."
-	patch $DPDK_DIR/drivers/net/i40e/i40e_rxtx.c $NGIC_DIR/patches/avx_not_suported.patch
+	patch $DPDK_DIR/drivers/net/i40e/i40e_rxtx_vec_avx2.c $NGIC_DIR/patches/v2-net-i40e-fix-avx2-driver-check-for-rx-rearm.diff
```
Comment (krsna1729) [inline install.sh]: Would this be better?
```
patch -d $DPDK_DIR -p1 < $NGIC_DIR/patches/v2-net-i40e-fix-avx2-driver-check-for-rx-rearm.diff
```

**Comment by krsna1729** [comment]
@ajamshed 1-2 line description to the commit message and update the first message in this conversation. Also change `Addresses` to `Fixes`. That way it will close the linked issue once we merge this. I was using `Addresses` for partial stuff. Thanks!

**Comment by krsna1729** [comment]
ok test please

**Comment by krsna1729** [comment]
ok to test

**Pull request merged**